prow/plugins/approve/approve_test.go

```diff
@@ -262,7 +262,7 @@ Approvers can cancel approval by writing ` + "`/approve cancel`" + ` in a commen
 			expectedComment: `[APPROVALNOTIFIER] This PR is **NOT APPROVED**
 
 This pull-request has been approved by:
-To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **cjwagner**
+To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **cjwagner** after the PR has been reviewed.
```

Comment:
should it say "after the PR has been reviewed and got lgtm label"? Also perhaps note "This PR was automatically assigned to one of the reviewers in the area for review." So together:

```suggestion
This PR was assigned to one of the reviewers in the area automatically. To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **cjwagner** after the PR has been reviewed  and got lgtm label.
```

Reply:
> This PR was assigned to one of the reviewers

Technically speaking, we "request reviews" from the reviewers instead of "assigning" them. 🙃 

👍 to  extending the note to mention that folks should be assigned only after the PR has the lgtm label.

Reply:
> This PR was assigned to one of the reviewers in the area automatically. To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **cjwagner** after the PR has been reviewed  and got lgtm label.

A more comprehensive rewording could look like:
_In this repository, pull requests are ready for merging once they are marked LGTM (looks good to me)
and then approved for merge.
The first review step is a technical check that the change is valid and matches the pull request description
and, if this passes, the reviewer will add the “lgtm” label. This bot has automatically selected
reviewers to perform the first step and has request their feedback.
The second step is approval for the pull request to merge, by a code owner.
Once this pull request has been reviewed and is labelled `lgtm`, please assign **cjwagner** to
request that they approve these changes._

Reply:
I'm very much against more verbose messages. In general the longer the message, the less likely it is to be read =/
Ideally it's short, direct and links out to a place like the contributor guide for more information.